pkg/crc/preflight/preflight_windows.go

```diff
@@ -19,39 +19,44 @@ var hypervPreflightChecks = []Check{
 		configKeySuffix:  "check-administrator-user",
 		checkDescription: "Checking if running in a shell with administrator rights",
 		check:            checkIfRunningAsNormalUser,
-		flags:            StartUpOnly,
+		fixDescription:   "crc should be ran in a shell without administrator rights",
```

Comment:
`should be ran`?

Why the passive voice wth a past tense?

Reply:
This used to be this way, I only c&p'ed what was removed in https://github.com/crc-org/crc/commit/1ef7f7b5ffc837835dfc0fe4a0865e8ddf2a164c
I'll change it. This should be "should be run", right?